toolkit/tools/imagecustomizer/main.go

```diff
@@ -18,15 +18,17 @@ import (
 var (
 	app = kingpin.New("imagecustomizer", "Customizes a pre-built CBL-Mariner image")
 
-	buildDir          = app.Flag("build-dir", "Directory to run build out of.").Required().String()
-	imageFile         = app.Flag("image-file", "Path of the base CBL-Mariner image which the customization will be applied to.").Required().String()
-	outputImageFile   = app.Flag("output-image-file", "Path to write the customized image to.").Required().String()
-	outputImageFormat = app.Flag("output-image-format", "Format of output image. Supported: vhd, vhdx, qcow2, raw.").Required().Enum("vhd", "vhdx", "qcow2", "raw")
-	configFile        = app.Flag("config-file", "Path of the image customization config file.").Required().String()
-	logFile           = exe.LogFileFlag(app)
-	logLevel          = exe.LogLevelFlag(app)
-	profFlags         = exe.SetupProfileFlags(app)
-	timestampFile     = app.Flag("timestamp-file", "File that stores timestamps for this program.").String()
+	buildDir                 = app.Flag("build-dir", "Directory to run build out of.").Required().String()
+	imageFile                = app.Flag("image-file", "Path of the base CBL-Mariner image which the customization will be applied to.").Required().String()
+	outputImageFile          = app.Flag("output-image-file", "Path to write the customized image to.").Required().String()
+	outputImageFormat        = app.Flag("output-image-format", "Format of output image. Supported: vhd, vhdx, qcow2, raw.").Required().Enum("vhd", "vhdx", "qcow2", "raw")
+	configFile               = app.Flag("config-file", "Path of the image customization config file.").Required().String()
+	rpmSources               = app.Flag("rpm-source", "Path to a RPMs tarball or a directory containing RPMs.").Strings()
+	disableBaseImageRpmRepos = app.Flag("disable-base-image-rpm-repos", "Disable the base image's RPM repos as an RPM source").Bool()
```

Comment:
Is this different from the existing `--disable-default-repos` and `--disable-upstream-repos`? Might want to align on a name for these if its doing the same thing.

Reply:
Yes and no. For image customizer, the default repo files are coming from the image file itself, not the toolkit.